packages/eas-cli/src/credentials/android/utils/keystore.ts

```diff
@@ -94,27 +117,49 @@ async function createKeystoreAsync(
       '-dname',
       `CN=,OU=,O=,L=,S=,C=US`,
     ]);
-
-    Analytics.logEvent(Event.ANDROID_KEYSTORE_CREATE, {
-      project_id: projectId,
-      step: KeystoreCreateStep.Success,
-      type: AndroidKeystoreType.Jks,
-    });
-
     return {
-      ...credentials,
+      ...keystoreParams,
       keystore: await fs.readFile(keystorePath, 'base64'),
       type: AndroidKeystoreType.Jks,
     };
-  } catch (error: any) {
-    Analytics.logEvent(Event.ANDROID_KEYSTORE_CREATE, {
-      project_id: projectId,
-      step: KeystoreCreateStep.Fail,
-      reason: error.message,
-      type: AndroidKeystoreType.Jks,
-    });
-    throw error;
   } finally {
     await fs.remove(keystorePath);
   }
 }
+
+interface KeystoreServiceResult {
+  keystoreBase64: string;
+  keystorePassword: string;
+  keyPassword: string;
+  keyAlias: string;
+}
+
+async function createKeystoreInCloudAsync(
+  keystoreParams: KeystoreParams,
+  { showKeytoolDetectionMsg = true } = {}
+): Promise<KeystoreWithType> {
+  if (showKeytoolDetectionMsg) {
+    Log.warn(`Detected that you do not have ${chalk.bold('keytool')} installed locally.`);
```

Comment:
```suggestion
    Log.log(`Detected that you do not have ${chalk.bold('keytool')} installed locally.`);
```

i think this is something that people don't necessarily need to be "warned" about, but rather just told about, and so `log` feels more appropriate than `warn`.

Reply:
I wasn't sure what log level to choose but if you also think this shouldn't be a warning then I'm ok with changing it.